decidim/decidim
# Add user and user groups profiles to Open Data files

#### :tophat: What? Why?
This PR adds user data available in api to the OpenData files. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #13530

#### Testing
1. Switch to the branch 
2. Start app, and export the Open data 
3. See the archive contains `users.csv` & `user_groups.csv`
4. See there is a proper Readme in place
5. 
:hearts: Thank you!
